android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -125,10 +129,14 @@ class ReactExoplayerView extends FrameLayout implements
         DrmSessionEventListener,
         AdEvent.AdEventListener {
 
+
     public static final double DEFAULT_MAX_HEAP_ALLOCATION_PERCENT = 1;
     public static final double DEFAULT_MIN_BACK_BUFFER_MEMORY_RESERVE = 0;
     public static final double DEFAULT_MIN_BUFFER_MEMORY_RESERVE = 0;
     public static final long DEFAULT_SKIP_DURATION = 15000;
+    public static final int DEFAULT_TV_CONTROLLER_PADDING = 50;
+    public static final int DEFAULT_TV_CONTROLLER_DURATION_FONT_SIZE= 18;
+    public static final int DEFAULT_TV_CONTROLLER_DURATION_WIDHT= 65;
```

Comment:
Small typo: 

`DEFAULT_TV_CONTROLLER_DURATION_WIDHT` to
`DEFAULT_TV_CONTROLLER_DURATION_WIDTH`